host/cluster-down.yaml

```diff
@@ -53,11 +53,11 @@ spec:
         get:
           url: https://localhost:6443/healthz
           insecureSkipVerify: true
-    - http:
```

Comment:
should we keep this check in here in addition to the end-to-end check via 6443?

as per discussion in the troubleshoot community meeting :)

Reply:
scenario: etcd is up but the K8s API server is down, we will at least see that in the output which is a better big picture?

Reply:
As we poke let's revert this one and working on this in a follow up PR